src/Warrior.java

```diff
@@ -0,0 +1,41 @@
+import java.util.List;
+
+/**
+ * Created by connorparrott123 on 12/6/2016.
+ */
+public class Warrior extends Character {
+    int weight;
+    int height;
+    String eyeColor;
+    String hairColor;
+
+    public Warrior(String name) {
+        super(name);
+        addWeapon(new Sword(name + "'s sword", "Plain Old Sword"));
+        weight = 230;
```

Comment:
weight, height, eyeColor, and hairColor should all come from parameters to this constructor, since not all Warriors are 230 kilograms etc